scapy/packet.py

```diff
@@ -927,6 +927,15 @@ def answers(self, other):
             return self.payload.answers(other.payload)
         return 0
 
+    def layers(self):
+        """returns a list of layer classes (including subclasses) in this packet"""  # noqa: E501
+        layers = []
+        l = self
```

Comment:
It’s useally a common practice to not use `l` as a variable (we have disabled this test in scapy because our code has tons), but it would be better not to create any.

In general, single letter variables are not liked that much. Especially with windows don’t, 1=l

How about `lay` ?